src/components/Password/Password.tsx

```diff
@@ -95,6 +95,12 @@ const Password = forwardRef<HTMLDivElement, PasswordProps>(
                             type="button"
                             onClick={() => {
                                 setIsHidden(prevValue => !prevValue);
+
+                                // set input focus
+                                const inputElement = document.querySelector(`input[id=${inputId}]`);
```

Comment:
I believe ideally we should use the `ref` that we are passing to the input instead of `querySelector`, but at the moment that won't work because the input's ref is actually being attached to the input wrapper instead of the actual input element (you can see it [in the `InnerInput` component](https://github.com/freenowtech/wave/blob/main/src/components/Input/InnerInput.tsx#L38)).

There is an issue to fix that (#169) but I'm not sure if we should wait for that to be resolved, since there may be users relying on the wrapper's ref and we may need to consider it a breaking change, I'd like to get other's opinions on this before making a decision